projects/questionset-editor-library/src/lib/services/tree/tree.service.spec.data.ts

```diff
@@ -6,6 +6,7 @@ export const treeNode = {
         "keywords": [
             "test"
         ],
+        "mode": true,
```

Comment:
Note:  `mode` should be renamed to meaningful name.

Do we need to store both `mode` and `evalMode` in question/questionset metadata for enabling server side evaluation ?

Reply:
done. We are storing evalMode for server side evaluation